MeasureAuthoringTool/mat-drupal
# MAT-938: New MAT User Request Form

## JIRA Tickets
MAT-938: New MAT User Request Form
MAT-965: [Prod] IE11 browser - Text overlapping on Training & Resources page 

## Description
Creates a webform to accept MAT User Registration requests. Does not save the webform submissions, only forwards them via email to be processed.

Update the secondary link on the homepage hero to point to a different location (one that still needs to be created on the production site).

## To Test
Import config.
Open the new form, and MailHog in a new tab (`localhost:8025`).
Fill out the form, submit the request.
Check that no data is saved about the submission in Drupal's DB.
Check that MailHog captured an email attempting to forward the request.
